app/policies/school_move_log_entry_policy.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class SchoolMoveLogEntryPolicy < ApplicationPolicy
```

Comment:
Since this is only used in the `SchoolMoveExport` I would be tempted to move it straight in to there, we can always pull it out in to a policy if we decided we need to show the log entries in more places.